.idea/.name

```diff
@@ -1 +1 @@
-Caffeine Project
```

Comment:
Checked some other public repos, and most do in fact ignore `.idea` dir and make import vs open the project.  I'll PR to remove it.